### DISCUSSION TITLE
Carousel view in a BlazorWebView in .net Maui

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#3446] [Q&A]
Hi, I´m starting to investigate if I can create a .net Maui app using BlazorWebView. 

And I would like to have  a horizontal scrolling (carousel) control like this here below. 

![image](https://user-images.githubusercontent.com/2386572/153901716-7bb36df2-4d7a-4540-a58e-a24bd6a12ff3.png)

I tried to use the Blazorise Carousel component but it doesn´t look or feel anywhere close to what you are custom to on a mobile app. E.g swiping was way off etc.

So is there something else I can use or can I manipulate the Carousel view somehow to behave better?

**Comment by stsrki** [comment]
I must admit I wasn't aware of this feature in Bootstrap so I have checked their documentation and it really works. Meaning we will also have to support it on our side. The problem here is that they use javascript and we have our own implementation that works in C# only. Could take some time to make it happen.

@David-Moreira what do you think?
